src/main/java/com/client/core/base/model/helper/AbstractHelper.java

```diff
@@ -22,7 +22,8 @@
 
     private final Logger log = Logger.getLogger(getClass());
 
-    private final BullhornData bullhornData;
+    @Autowired
+    private BullhornData bullhornData;
```

Comment:
Isn't it possible for us to inject this instead of autowiring it?

Reply:
@scriptom 
The annoying part is that the constructor for AbstractHelper is called in AbstractTriggerHelper, and AbstractScheduledTaskHelper.

That means BullhornData needs to be injected in those constructors as well... And those constructors are called in all of the implementations of the abstract helpers.

Reply:
I see. Let's leave it like it is now instead of injecting. Payoff's not good enough.